packages/addon-info/src/index.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from "react";
 import { useChannel, useState } from "@storybook/client-api";
 import { TestComponent } from "./components/TestComponent";
+import {REQUEST_PROPS_CHANGE} from "./constants";
 
 export const withTestComponent = (storyFn: () => any) => {
   const [selected, setSelected] = useState<boolean>(true);
 
   const newSelect = !selected;
 
   useChannel({
-    REQUEST_PROPS_CHANGE: () => setSelected(newSelect)
-  });
+    [REQUEST_PROPS_CHANGE]: () => setSelected(newSelect)
```

Comment:
Before the change, the event name being listened to was `'REQUEST_PROPS_CHANGE` string instead of `REQUEST_PROPS_CHANGE` constant value which is actually `'addon/request-info-change'`